downstream/modules/platform/proc-configuring-the-metrics-utility.adoc

```diff
@@ -18,50 +20,65 @@ See link:https://www.redhat.com/sysadmin/linux-cron-command[How to schedule jobs
 
 .Procedure
 
-. In the cron file, set the following variables to ensure `metrics-utility` gathers the relevant data. To open the cron file for editing, run: 
-+
-[source, ]
-----
-crontab -e
-----
-+
-. Specify the following variables to indicate where the report is deposited in your file system:
+. Create two scripts in your user's home director in order to set correct variables to ensure that `metrics-utility` gathers all relevant data.
```

Comment:
Nitpick, feel free to disagree. Is there a particular reason why metrics-utility is in monospace formatting in some instances and not in the rest? Consider keeping it consistent across.